### DISCUSSION TITLE
Incorrect Thumbnails for some videos

### DISCUSSION THREAD

**Discussion by thisIsNotTheFoxUrLookingFor** [photoprism/photoprism#4472] [Q&A] [answered]
I create a brand new instance, I delete the entire storage (and database) folder to ensure no artefacts remain.

I copy the pics and vids into the import folder, I then click import in the UI and it imports the pictures and videos into the originals folder.

It builds thumbnails for them, but some videos are getting thumbnails from other pictures that have absolutely nothing to do with the video.

I have rebuilt PhotoPrism several times and the exact same videos cause the same error every time.

Example:

Here is the thumbnail generated:
![Screenshot 2024-08-21 135706](https://github.com/user-attachments/assets/feffd2f5-772e-471b-9e81-3fb5c042c7b1)

Here is the actual video, it is of a fire bombing helicopter and it does not contain a map of Australia.
![image](https://github.com/user-attachments/assets/0a278b2f-13a3-474f-a498-ea2644b7dd33)

How does this occur?

**Comment by lastzero** [comment]
Most likely images with the same name as the video files, so they get stacked: https://docs.photoprism.app/user-guide/organize/stacks/#for-what-reasons-can-files-be-stacked

If you want to file an official bug report, please use the template provided and make sure that the feature is not working as documented. Thank you!

**Reply by thisIsNotTheFoxUrLookingFor** [reply]
> Most likely images with the same name as the video files, so they get stacked: https://docs.photoprism.app/user-guide/organize/stacks/#for-what-reasons-can-files-be-stacked
> 
> If you want to file an official bug report, please use the template provided and make sure that the feature is not working as documented. Thank you!

Ok but I disabled stacking before importing the files, should it still stack?

**
![image](https://github.com/user-attachments/assets/de9e28c2-9768-4369-9248-6775b2cf14a1)
**
@lastzero is this expected with stacking disabled or should I file a bug?

**Reply by lastzero** [reply]
Please see the documentation we provide :)

**Reply by thisIsNotTheFoxUrLookingFor** [reply]
> Please see the documentation we provide :)

Sure but the documentation does not answer my question. I have disabled stacking, and clearly it is stacking, so should I be reporting a bug about the stacking that is occurring when stacking is disabled?

**Comment by thisIsNotTheFoxUrLookingFor** [comment, marked as answer]
@lastzero 

I guess it is the case that condition 1 happens always and cannot be disabled?

that is very annoying, I have to go and manually unstack these videos.

![image](https://github.com/user-attachments/assets/2d0194fd-587e-44d8-8e58-1c9ff60e45ad)

**Comment by thisIsNotTheFoxUrLookingFor** [comment]
Also these files still got stacked different names and they are not even similar/related at all
![image](https://github.com/user-attachments/assets/6ebf0a2c-94cd-424d-9343-8519f7f208dc)

**Reply by lastzero** [reply]
We are currently developing new features that our users are waiting for, so we have very limited time to help. I recommend checking related issues as there's also other reasons this could happen e.g. they were stacked before you changed the settings (maybe the UID got restored from a sidecar YAML file?) or there's a database/filesystem encoding issue, for example:
- https://github.com/photoprism/photoprism/issues/4309

**Comment by thisIsNotTheFoxUrLookingFor** [comment]
Okies no worries thanks for the help so far anyways.
